### ISSUE TITLE
Missing focus-visible state on Firefox and Safari in `<quiet-toggle-tag>` and `<quiet-slide-activator>`

### ISSUE THREAD

**Issue by sunnywalker** [quietui/quiet#31]
## Describe the bug

When using keyboard navigation, the focused element indicator (focus ring) is lost on `<quiet-toggle-tag>` and `<quiet-slide-activator>` in Firefox and Safari.

## How can I reproduce the bug?

1. Go to https://quietui.org/docs/components/toggle-tag
2. Hit tab until the skip to content link is displayed and hit Enter
3. Tab past the "checkboxes", "switches", and "checkbox groups" links which properly show focus state.
4. Tabbing through the toggle tags samples, such as the Amenities section, no focus outline or other indicator is shown in Firefox and Safari.

The same happens with `<quiet-slide-activator>` found at https://quietui.org/docs/components/slide-activator.

## Affected Browsers / Operating Systems

 - Browsers (include versions): Firefox 144, Safari 26.0.1
 - OS: tested Firefox on Mac and Windows, Safari on Mac

## Additional Information

Both elements operate as expected, but the focus ring just doesn't show, so it's hard to know which amenity is selected (in the Amenities sample).

It doesn't matter whether a toggle tag is on or off--the focus ring is missing from both states.
